tests/Oryx.RuntimeImage.Tests/Php-fpm/PhpFpmImageTest.cs

```diff
@@ -149,5 +149,32 @@ public void PhpFpmRuntimeImage_Contains_VersionAndCommit_Information(string vers
                 },
                 result.GetDebugInfo());
         }
+
+        [Theory]
+        [InlineData("8.0")]
+        [InlineData("7.4")]
+        [InlineData("7.3")]
+        [InlineData("7.2")]
+        [InlineData("7.0")]
+        public void Redis_IsInstalled(string imageTag)
```

Comment:
add some of these tests in phpfpm as well as we have added redis for both fpm based image and apache baded image

Reply:
unit tests have been added now